.github/workflows/test.yml

```diff
@@ -106,6 +106,6 @@ jobs:
         uses: actions/setup-node@v2
       - name: Npm Install
         if: steps.cache.outputs.cache-hit != 'true'
-        run: npm install
+        run: npm ci
       - name: Run diff check for the breakUpAriaJSON script
-        run: node scripts/breakUpAriaJSON.js && git diff --exit-code -- src
+        run: node scripts/breakUpAriaJSON.js && git diff --exit-code
```

Comment:
@nschonni can you revert this change? The diff check should only happen on the src dir, not the whole repo

Reply:
Just the one change revert.

Reply:
I can revert this, but this was only required because you weren't using `ci` for the install before. The lockfile won't be changing any more

Reply:
It performs the intended check explicitly, so no harm here.